Microsoft.DotNet.Interactive.Tests/LanguageServiceSerializationTests.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using FluentAssertions;
+using FluentAssertions.Execution;
+using Microsoft.DotNet.Interactive.LanguageService;
+using Newtonsoft.Json.Linq;
+using Xunit;
+
+namespace Microsoft.DotNet.Interactive.Tests
+{
+    public class LanguageServiceSerializationTests
+    {
+        private T Parse<T>(string json)
+        {
+            var jo = JObject.Parse(json);
+            return jo.ToLspObject<T>();
```

Comment:
This will use default serialization settings. What we're doing in `Microsoft.DotNet.Interactive.Server` is encapsulating the serialization (e.g. no explicitly dependency on `NewtonSoft.Json`) and considering serialization and deserialization as part of our API and contract.

Reply:
LSP types have a strict expectation on member casing that `ToLspObject()` honors; it wouldn't be valid anywhere to serialize it any differently.